### DISCUSSION TITLE
V-Button icon prop not working in custom extension

### DISCUSSION THREAD

**Discussion by riccardostokker** [directus/directus#15773] [Archive]
Hello everyone!
Just got into Directus, it's really good, but I ran into an issue.
I'm getting started in the development of a custom extension and I'm trying to use the `v-button` component as specified in the wiki:
```
      <v-button icon>
        <v-icon name="fullscreen"/>
      </v-button>
```
Should work out of the box right? However the button with the fullscreen icon has a background and padding.

![image](https://user-images.githubusercontent.com/62010554/192612020-e0798547-a307-4751-a934-83450d417b84.png)

I'm using:
- @directus/extensions-sdk v9.18.1
- Directus 9.18.1

The plugin was rebuilt after the changes and Directus was restarted after that.

Another quick question: can I use a framework like TailwindCSS while developing a Directus extension? Is it reccommended?

Thank you very much!
Riccardo

**Comment by rijkvanzanten** [comment]
> Should work out of the box right? However the button with the fullscreen icon has a background and padding.

That's expected, as the `v-button` is a styled component from the app 🙂 

>  can I use a framework like TailwindCSS while developing a Directus extension? Is it reccommended?

You theoretically could, as a custom extension is just a Vue component (so you can include basically whatever you want). That being said, I wouldn't necessarily recommend doing so, as it would increase the bundle size of your extension quite a bit, and won't really fit the rest of the directus styling all that well
